content/id/docs/concepts/overview/working-with-objects/kubernetes-objects.md

```diff
@@ -0,0 +1,109 @@
+---
+title: Memahami Konsep Objek-Objek yang Ada pada Kubernetes
+content_template: templates/concept
+weight: 10
+card: 
+  name: concepts
+  weight: 40
+---
+
+{{% capture overview %}}
+Laman ini menjelaskan bagaimana objek-objek Kubernetes direpresentasikan di dalam API Kubernetes, 
+dan bagaimana kamu dapat merepresentasikannya di dalam format `.yaml`.
+{{% /capture %}}
+
+{{% capture body %}}
+## Memahami Konsep Objek-Objek yang Ada pada Kubernetes
+
+*Objek-Objek Kubernetes* adalah entitas `persisten` di dalam sistem Kubernetes. 
```

Comment:
I would prefer doing italic for imported english words/terms, over backtick

we should follow this guideline for backtick (otherwise it will get harder and harder to read, as per my comments on #14117)
https://kubernetes.io/docs/contribute/style/style-guide/#inline-code-formatting

I don't think our Indonesian styleguide will differ from this. IMO our Indonesian styleguide should just be original english styleguide + italic imported english words (+ Indonesian specifics if any)

wdyt?

this comment applies for all imported words on backticks

Reply:
agreed..